### PR TITLE
Initial commit of material3 base template

### PR DESCRIPTION
**Pull request by johnshea**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/Theme.Amphibians"
+        tools:targetApi="31">
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml]: 33

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <activity
+            android:name=".MainActivity"
+            android:exported="true"
+            android:label="@string/app_name"
+            android:theme="@style/Theme.Amphibians">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
```
Comment (kkuan2011) [inline app/src/main/AndroidManifest.xml]: nit: can remove empty line (though I know it appears in the template automatically)

File: app/src/main/java/com/example/amphibians/ui/theme/Theme.kt [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    secondary = PurpleGrey80,
+    tertiary = Pink80
+)
+
+private val LightColorScheme = lightColorScheme(
+    primary = Purple40,
+    secondary = PurpleGrey40,
+    tertiary = Pink40
+
+    /* Other default colors to override
```
Comment (kkuan2011) [inline app/src/main/java/com/example/amphibians/ui/theme/Theme.kt]: Can you remove unused code?

File: app/src/main/java/com/example/amphibians/ui/theme/Type.kt [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+// Set of Material typography styles to start with
+val Typography = Typography(
+    bodyLarge = TextStyle(
+        fontFamily = FontFamily.Default,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp,
+        lineHeight = 24.sp,
+        letterSpacing = 0.5.sp
+    )
+    /* Other default text styles to override
```
Comment (kkuan2011) [inline app/src/main/java/com/example/amphibians/ui/theme/Type.kt]: same here, remove unused code

**Review by kkuan2011 — CHANGES_REQUESTED** [review]
LGTM aside from these comments

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (johnshea) [reply app/src/main/AndroidManifest.xml]: Done.

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (johnshea) [reply app/src/main/AndroidManifest.xml]: Done.

File: app/src/main/java/com/example/amphibians/ui/theme/Type.kt (hunk 4 above)
Comment (johnshea) [reply app/src/main/java/com/example/amphibians/ui/theme/Type.kt]: Done.

**Review by kkuan2011 — APPROVED** [review]

**Pull request closed without merging**